data/human/human missions.txt

```diff
@@ -5416,3 +5416,346 @@ mission "Deal Change 2"
+			`	You can barely restrain yourself from making a snappy remark, but you have learned that it's best not to antagonize your passengers.`
+	on enter
+		conversation
+			`You hear hurling and splashing noises in the background. It seems your passenger threw up, not uncommon for first-timers.`
+			choice
+				`	(Go and assist her.)`
+					goto assist
+				`	(Mind my own business.)`
+					goto stay
+
+			label assist
+			`	You go to the passenger compartments to find her standing in the middle of the room, covered in puke. Somehow she also managed to paint all four walls of her cabin with vomit.`
+			choice
+				`	"Are you enjoying the ride, ma'am?"`
+					goto nothing
+				`	(Say nothing.)`
+					goto nothing
+
+			label stay
+			`	After a couple of minutes, your passenger busts into the cockpit, puke dripping from her clothes.`
+
+			label nothing
+			`	"Look at this mess you made! Whackos like you should never be allowed to drive around! This is a spaceship, not a space rodeo!"`
+			`	You try to tell her about the nature of hyperspace jumps, but that only results in a volley of further insults.`
+	on enter
+		conversation
+			`You hear muffled noises behind you. You move to the back of the cockpit to see your passenger throwing a fit, beating on the door in anger. "I demand to speak to the captain!"`
+			choice
+				`	(Talk with her.)`
+					goto talk
+				`	(Restrain her.)`
+					goto restrain
+
+			label talk
+			`	You open the door, but she cuts you off before you have a chance to say something.`
+			`	"Idiot! You flying space morons can't even read a map! We are nowhere near <planet>! Are you trying to kidnap me? I'll get the Navy to put you into prison for this!" You try to speak, but can't get a word in edgeways.`
+
+			label restrain
+			`	You call your crew, and together you manage to detain her. She is now locked up in her cabin, but her screams still echo in your head. This will be a long day.`
+			`	You decide to drop her off on the first planet you land on.`
+	on complete
+		conversation
+			`You are glad to finally see your passenger leaving, but then you realize she has't paid you yet. You decide to confront her about it.`
+			`	"You expect ME to pay after all that? You are lucky I'm not suing you for damages!" She stops for a moment, then throws a credit chip at you. "Fine, take this, but don't bother me again!" She stomps off angrily.`
+			`	You look at the credit chip she had so generously given you. It's worth... 10 credits. Oh well.`
+		payment 10
+	on abort
+		conversation
+			branch landed
+				has "flagship landed"
+			`You decide you had enough of her constant fits, and inform her that her journey is over.`
+			`	As you listen with grim satisfaction to the air hissing out of the opening airlock, you take comfort in the knowledge that you at least got the bunk space back.`
```

Comment:
This is the unspoken truth about ES. When you abort a mission mid-flight, you get the cargo and bunk space back - and there isn't a lot you can do with those people in the middle of deep space.

Reply:
That's true, yes. I just meant I'm not so this mission on it's own is the place to officially canonize/denote that as literal within the abstraction of the game. Like, if we're going to recognize that as a literal occurrence, there should be some kind of specific consequences to address that for spacing Katya or Ijs (for example) beyond -just- failing the campaign, and we may even want to address consequences for repeatedly doing so on this and other random jobs.

If no one else has a problem with it being introduced here specifically, I don't consider it any kind of block, just musing about whether or not adding it back here alongside also accounting for it elsewhere may be better as a separate thing with more focused scrutiny.

Reply:
[From Pointed](https://github.com/endless-sky/endless-sky/pull/5781#discussion_r593755478):

>Think about how this would work in real life: if you directly and negligently facilitated the death of 5-10 innocent yet stupid people what's gonna happen? A 10k fine? Or imprisonment for a decade or more?
>
>I know games aren't real life, but you can't bend the rules of physics or how human societies work too much, or else it breaks the player out of the illusion that the game is crafting.

Also, just because something happens in gameplay doesn't mean that it is codified in story. See Derpy's comments on the no-mercy Navy in Hai Revealed.

Reply:
I would like to have booting people out midflight be acknowledged in some way, but there definitely should be actual consequences if we're going to do that.